components/HomeChapters/index.tsx

```diff
@@ -15,7 +15,7 @@ const HomeChapters: React.FC<Props> = ({ chapters }) => {
     React.useEffect(() => {
         if (navigator) {
             navigator.geolocation.getCurrentPosition(
-                (pos: GeolocationPosition) => {
+                (pos: Position) => {
```

Comment:
`Position` breaks `yarn build` and should be left as `GeolocationPosition`. Should work with the updated version of TypeScript

Reply:
Will do, I'll revert it back to `GeolocationPosition`